osbs/schemas/container.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
What was the reason for the jsonschema version bump?

I don't mind mind it too much if it doesn't break anything, but I would prefer to have it in a separate commit.

Reply:
it is to allow the deprecated key in the schema. It's not available in draft 6

Reply:
moved to separate commit

Reply:
Ohh, that makes sense, I didn't even notice the `deprecated` key before :smile: